assets/src/edit-story/components/header/title.js

```diff
@@ -79,15 +103,17 @@ function Title() {
   }
 
   return (
-    <Input
-      ref={setTitleInput}
-      value={title}
-      type={'text'}
-      onBlur={handleBlur}
-      onChange={handleChange}
-      placeholder={__('Add title', 'web-stories')}
-      aria-label={__('Story title', 'web-stories')}
-    />
+    <HighlightContainer {...highlight}>
+      <Input
+        ref={setTitleInput}
+        value={title}
+        type={'text'}
```

Comment:
We don't need string interpolation here
```suggestion
        type="text"
```